core/domain/stats_jobs_one_off.py

```diff
@@ -802,6 +802,20 @@ def entity_classes_to_map_over(cls):
     @classmethod
     def require_non_negative(
             cls, exp_id, exp_version, property_name, value, state_name=None):
+        """Yields the Negative count of the exploration according
```

Comment:
I do not understand what "Yields the Negative count" means, or why "Negative" is capitalized.

Instead, I suggest something like: "Yields an error message if the count associated with the given property_name in the value dict is is negative."